dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinRefBoxController.java

```diff
@@ -348,17 +348,19 @@ private List<ClarinFeaturedServiceLink> mapFeaturedServiceLinks(List<MetadataVal
     }
 
     /**
-     * Remove the unnecessary parts from the output. It is removing the XML header and the <bib:bibtex> tag.
+     * Remove the unnecessary parts from the output.
      */
     private String updateOutput(String type, String output) {
         try {
             if (StringUtils.equals(type, BIBTEX_TYPE)) {
-                return removeBibtexTag(output);
+                // Remove the XML header tag and the <bib:bibtex> tag from the string.
+                return getXmlTextContent(output);
             } else {
+                // Remove the XML header tag from the string.
                 return removeXmlHeaderTag(output);
             }
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error("Cannot update the xml string for citation because of: " + e.getMessage());
```

Comment:
you will miss stack trace here. Might be ok, might be not, but if so, do this intentionally please.